pkg/oc/cli/cmd/wrappers.go

```diff
@@ -480,7 +480,7 @@ func NewCmdRun(fullName string, f *clientcmd.Factory, in io.Reader, out, errout
 	cmd := kcmd.NewCmdRunWithOptions(f, opts, in, out, errout)
 	cmd.Long = runLong
 	cmd.Example = fmt.Sprintf(runExample, fullName)
-	cmd.SuggestFor = []string{"image"}
+	cmd.SuggestFor = []string{"start"}
```

Comment:
start doesn't seem to be informative here. I'd just drop this suggestion entirely.

Reply:
Alright, updated. I have already squashed, as this is small PR.